app/models/contrato.rb

```diff
@@ -0,0 +1,44 @@
+require 'httparty'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.